config/printer-creality-ender3-s1-2021.cfg

```diff
@@ -0,0 +1,128 @@
+# This file contains pin mappings for the stock 2021 Creality Ender 3
+# S1. To use this config, during "make menuconfig" select the
+# STM32F103 with a "28KiB bootloader" and serial (on USART1 PA10/PA9)
+# communication.
+
+# If you prefer a direct serial connection, in "make menuconfig"
+# select "Enable extra low-level configuration options" and select
+# serial (on USART3 PB11/PB10), which is broken out on the 10 pin IDC
+# cable used for the LCD module as follows:
+# 3: Tx, 4: Rx, 9: GND, 10: VCC
+
+# Flash this firmware by copying "out/klipper.bin" to a SD card and
+# turning on the printer with the card inserted. The firmware
+# filename must end in ".bin" and must not match the last filename
+# that was flashed.
+
+# See docs/Config_Reference.md for a description of parameters.
+
+[stepper_x]
+step_pin: PC2
+dir_pin: PB9
+enable_pin: !PC3
+microsteps: 16
+rotation_distance: 40
+endstop_pin: !PA5
+position_endstop: 0
+position_max: 220
+position_min: -5
+homing_speed: 50
+
+[stepper_y]
+step_pin: PB8
+dir_pin: PB7
+enable_pin: !PC3
+microsteps: 16
+rotation_distance: 40
+endstop_pin: !PA6
+position_endstop: 0
+position_max: 220
+position_min: -5
+homing_speed: 50
+
+[stepper_z]
+step_pin: PB6
+dir_pin: !PB5
+enable_pin: !PC3
+microsteps: 16
+rotation_distance: 8
+endstop_pin: probe:z_virtual_endstop
+position_max: 270
+position_min: -3
+
+[extruder]
+step_pin: PB4
+dir_pin: PB3
+enable_pin: !PC3
+microsteps: 16
+gear_ratio: 3.5:1
+rotation_distance: 20
```

Comment:
I have and s1 pro and I tuned today the rotation distance, the value that I obtain according with klipper documentation was 25.75.
20 is over extruding
https://www.klipper3d.org/Rotation_Distance.html.

Reply:
20 should be underextruding if you got 25.75, do some test prints, I have someone using the config and they have no issues with overextruding, https://cdn.discordapp.com/attachments/970011305975095316/970345455739351040/IMG_0118.jpg